WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -943,11 +946,28 @@ public void onSiteChanged(OnSiteChanged event) {
         if (site != null) {
             mSelectedSite = site;
         }
+        if (getMySiteFragment() != null) {
+            getMySiteFragment().onSiteChanged(site);
+        }
     }
 
     @SuppressWarnings("unused")
     @Subscribe(threadMode = ThreadMode.MAIN)
     public void onSiteRemoved(OnSiteRemoved event) {
         handleSiteRemoved();
     }
+
+    @Override public void onPositiveClicked(@NonNull String instanceTag) {
+        MySiteFragment fragment = getMySiteFragment();
+        if (fragment != null) {
+            getMySiteFragment().onPositiveClicked(instanceTag);
```

Comment:
This statement is still referencing `getMySiteFragment()` rather than `fragment` like the `onNegativeClicked` method below.

Reply:
Sorry, I must have missed that. 👍